src/Agent/NewRelic/Agent/Core/AgentHealth/AgentHealthReporter.cs

```diff
@@ -828,5 +829,14 @@ private void ReportIfLoggingDisabled()
                 ReportSupportabilityCountMetric(MetricNames.SupportabilityLoggingFatalError);
             }
         }
+
+        private void ReportIfInstrumentationIsDisabled()
+        {
+            var ignoredCount = _configuration.IgnoredInstrumentation.Count();
+            if (ignoredCount > 0)
+            {
+                ReportSupportabilityGaugeMetric(MetricNames.SupportabilityIgnoredInstrumentation, ignoredCount);
```

Comment:
So we're only reporting a count of ignored instrumentation, not a list of what's actually being ignored? I'm wondering if it would be helpful to report the details as supportability metrics?

Reply:
That would definitely be helpful, but it's not possible with the current Supportability metrics. We would need to create a separate metric for each assembly and class combination.